sympy/ntheory/factor_.py

```diff
@@ -2129,3 +2129,59 @@ def is_mersenne_prime(n):
 
     r, b = integer_log(n + 1, 2)
     return b and r in MERSENNE_PRIME_EXPONENTS
+
+
+def abundance(n):
+    return divisor_sigma(n, 1) - 2 * n
+
+
+def is_abundant(n):
+    """Returns True if ``n`` is an abundant number, else False.
+
+    A abundant number is smaller than the sum of its positive proper divisors.
+
+    Examples
+    ========
+
+    >>> from sympy.ntheory.factor_ import is_abundant
+    >>> is_abundant(20)
+    True
+    >>> is_abundant(15)
+    False
+
+    References
+    ==========
+
+    .. [1] http://mathworld.wolfram.com/AbundantNumber.html
+
+    """
+    n = as_int(n)
+    if is_perfect(n):
+        return False
+    return n % 6 == 0 or bool(abundance(n) > 0)
```

Comment:
I just had a look here. Can we not add a few more facts:
 - Prime numbers are not abundant
 - odd numbers < 945 are not abundant
 - Multiples of 20 are abundant

Reply:
Multiples of 6 except 6, multiples of 20,42,56,66,70 and many more are abundant.
If we get a abundant number which is not the multiple of previous abundant number. Then, multiple of that abundant number will always be abundant number.

Reply:
@Yathartha22 @smichr 
Should we add these facts?

Reply:
> Should we add these facts?

I think we just stick with the low hanging fruit here: 1/6 numbers benefit from the 6 test. Very few benefit from the other tests.